pkgs/development/python-modules/pyrdfa3/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchpatch
+, isPy27
+, rdflib
+, html5lib
+}:
+
+buildPythonPackage rec {
+  pname = "pyRdfa3";
```

Package names should be normalized according to PEP503, which in this case means applying lower case.

```suggestion
  pname = "pyrdfa3";
```

---

Oops, I'd missed that one.